snippets/cpp/cpp.json

```diff
@@ -266,5 +265,54 @@
             "}"
         ],
         "description": "A standard main function for a C++ program"
+    },
+    "#guard": {
+      "prefix": "#guard",
+      "body": [
+        "#ifndef INCLUDE_${TM_DIRECTORY/.*[\\\/](.*)/${1:/upcase}/}_${TM_FILENAME_BASE/(.*)/${1:/upcase}/}_H_",
+        "#define INCLUDE_${TM_DIRECTORY/.*[\\\/](.*)/${1:/upcase}/}_${TM_FILENAME_BASE/(.*)/${1:/upcase}/}_H_",
+        "",
+        "$0",
+        "",
+        "#endif  // INCLUDE_${TM_DIRECTORY/.*[\\\/](.*)/${1:/upcase}/}_${TM_FILENAME_BASE/(.*)/${1:/upcase}/}_H_"
+      ],
+      "description": "header guard. format :\n\tINCLUDE_<currdir_name>_<currfile_name>_H_"
+    },
+    "sca": {
```

Comment:
For these next few snippets, could you add in a small description instead of just the empty string? It would help with consistency of the snippets :grin: